recipes/cryolobm/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - matplotlib-base >=3.4.2,<3.5
     - numpy >=1.16.0,<1.19.0
     - pystardb >=0.3.0
-    - pandas ==1.1.4
+    - pandas <=1.4.4
```

Comment:
Should probably raise this upstream b/c it is in the package metadata.

Reply:
ok, I will do that.